### PR TITLE
Fixed issues related to Deprecation warning due to invalid escape seq…

### PR DESCRIPTION
**Pull request by vishal-purohit**
Before:
Screenshot from 2020-08-30 08-58-13

After:
Screenshot from 2020-08-30 09-17-56

Please let me know if any further changes are required

### CODE REVIEW

**Comment by vishal-purohit** [comment]
Please merge the pull request

File: deepchem/utils/geometry_utils.py [hunk 1]
```diff
@@ -65,8 +65,8 @@ def generate_random_unit_vector() -> np.ndarray:
   Citation: http://mathworld.wolfram.com/SpherePointPicking.html
 
   Pseudocode:
-    a. Choose random theta \element [0, 2*pi]
-    b. Choose random z \element [-1, 1]
+    a. Choose random theta \\element [0, 2*pi]
+    b. Choose random z \\element [-1, 1]
```
Comment (nd-02110114) [inline deepchem/utils/geometry_utils.py:69]: About this lines, it is better to do the same modifications like `generate_random__unit_vector` of `deepchem/feat/rdkit_grid_featurizer.py`. 

```
r"""Generate a random unit vector on the sphere S^2.
```

**Review by nd-02110114 — COMMENTED** [review]
Thank you for contribution! I reviewed!

**Comment by nd-02110114** [comment]
This issue was fixed by #2199

**Pull request closed without merging**